### PR TITLE
build alpine base image

### PR DESCRIPTION
**Pull request by BenRedmondBenham**
Could you push an alpine base image version?

Then you could support more complex commands like this:

```
grpcurl -plaintext \
             -H "Authorization: Bearer $$(cat /var/run/secrets/tokens/vault-token)" \
             localhost:80 \
             example.v1.Service/Func
```

This works with a common auth pattern on kubernetes where auth tokens are mounted into a volume on a pod.

```
apiVersion: v1
kind: Pod
metadata:
  name: nginx
spec:
  containers:
  - image: nginx
    name: nginx
    volumeMounts:
    - mountPath: /var/run/secrets/tokens
      name: vault-token
  serviceAccountName: build-robot
  volumes:
  - name: vault-token
    projected:
      sources:
      - serviceAccountToken:
          path: vault-token
          expirationSeconds: 7200
          audience: vault
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,6 +16,15 @@ RUN go build -o /grpcurl \
     -ldflags "-w -extldflags \"-static\" -X \"main.version=$(cat VERSION)\"" \
     ./cmd/grpcurl
 
+FROM alpine:3 as alpine
```
Comment (dragonsinth) [inline Dockerfile:19]: what is alpine:3 in particular?

**Comment by dragonsinth** [comment]
seems reasonable to me

File: Dockerfile (hunk 1 above)
Comment (BenRedmondBenham) [reply Dockerfile:19]: @dragonsinth  This will use the latest stable version of alpine without jumping up major versions.

**Review by dragonsinth — APPROVED** [review]
Looks like copy/paste from the `scratch` version so LGTM

**Pull request merged**